Dockerfile.pai

```diff
@@ -0,0 +1,13 @@
+FROM ibmcom/powerai:1.6.0-base-ubuntu18.04-py3
+SHELL ["/bin/bash", "-c"]
+
+WORKDIR /workspace
+RUN mkdir assets
+
+COPY requirements.txt /workspace
```

Comment:
How about change this line to `COPY . .` and remove the last line of this file?

Reply:
Yep, that works. Changed.